pkgs/development/libraries/libfprint-2-tod1-goodix/default.nix

```diff
@@ -25,6 +25,8 @@ stdenv.mkDerivation {
     cp lib/udev/rules.d/60-libfprint-2-tod1-goodix.rules "$out/usr/lib/udev/rules.d/"
   '';
 
+  passthru.driverPath = "/usr/lib/libfprint-2/tod-1";
```

Comment:
Any particular reason why this drive is installing to `$out/usr/lib`?

Reply:
@sternenseemann is there a documented convention or RFC?

Reply:
Didn't find a better passage, but <https://nixos.org/manual/nixpkgs/unstable/#multiple-output-file-type-groups> lines out the convetion pretty decently.

I think it makes sense to stick to that since a lot of stuff expects this (for example man pages in `$out/usr/share/man` wouldn't be accessible, `$out/usr/bin` isn't added to `PATH`…).

Reply:
Pushed this as a separate commit. Thanks for review!